tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/builder/BallerinaASTModelBuilder.java

```diff
@@ -78,21 +80,33 @@ public BallerinaASTModelBuilder() {
         }
     }
 
+    /**
+     * Check whether action statement is ready to be processed.
```

Comment:
I suppose there should be a new line after the method description.

Reply:
Please refer to "Format of a Doc Comment" at
http://www.oracle.com/technetwork/articles/java/index-137868.html

Reply:
This is fixed now.